TamoStudyStream/src/MainGUI.java

```diff
@@ -155,7 +155,17 @@ private void initFrame() {
 		this.setTitle("TamoStudyStream v0.1");
 		this.setIconImage(new ImageIcon(getClass().getClassLoader().getResource("ICON.png")).getImage());
 		this.setSize(500,500);
-		this.setLocationRelativeTo(null);
+    if (isWindowsOrMac())
```

Comment:
We can probably just do something like

if (not System.getProperty("os.name").startsWith("Linux")) then this.setLocationRelativeTo(null)

Reply:
probably, i didnt look at the strings that os.name outputted. My fear is that there is a lot of different kinds of linux and they might not contain the string "linux". I'll try it tho

Reply:
http://lopica.sourceforge.net/os.html

This link contains the potential outputs of "System.getProperties("os.name")";. We _could_ do "this.setLocationRelativeTo(null);" for just linux, however this could cause compatibility issues with SunOS, Digital Unix, and AIX distributions of Linux.

Reply:
Send me an executable jar. We may be able to simplify the code further. Perhaps if( windows or Mac) then set location.

Reply:
[Uploading TamoStreamBuild.zip…]()